go.mod

```diff
@@ -12,3 +12,17 @@ require (
 	github.com/patrickmn/go-cache v2.1.0+incompatible // indirect
 	github.com/redis/go-redis/v9 v9.0.5 // indirect
 )
+
+retract (
```

Comment:
this is noteworthy

Reply:
Nice and tricky I see that you left 0.6.5 out since we know that is where union is. But they may also be testing 6.6. somewhere. I guess it is ok to force them to shift to 0.6.8

Reply:
v0.6.6, v0.6.7, and v0.7.0 all fail our sdk check, are you ok that we'll be publishing and bumping everyone to 0.7.1? (not 0.6.8)